Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -0,0 +1,269 @@
+ *          if (splitProps.focusableProps !== undefined) {
+ *             this._focusable.setNativeProps(splitProps.focusableProps)
+ *          }
+ *          if (splitProps.childProps !== undefined) {
+ *             this._child.setNativeProps(splitProps.childProps);
+ *          }
+ *      }
+ *   }
+ *
+ * @providesModule FocusableWindows
+ */
+'use strict';
+
+var PropTypes = require('prop-types');
+var React = require('React');
+var ReactNative = require('ReactNative');
+var ViewPropTypes = require('ViewPropTypes');
+var requireNativeComponent = require('requireNativeComponent');
+var UIManager = require('UIManager');
+const flattenStyle = require('flattenStyle');
+
+class FocusableWindows extends React.Component {
+
+  static propTypes = {
+    ...ViewPropTypes,
+
+    /**
+     * Controls whether the view is a tab stop. Useful for buttons and other
+     * controls that can be focused.
+     *
+     * @platform windows
+     */
+    isTabStop: PropTypes.bool,
+
+    /**
+     * Sets the tab index for the view.
+     *
+     * @platform windows
+     */
+    tabIndex: PropTypes.number,
+
+    /**
+     * Sets the keyboard navigation mode for the view.
+     *
+     * @platform windows
+     */
+    tabNavigation: PropTypes.oneOf([
+      'local',
+      'cycle',
+      'once'
+    ]),
+
+    /**
+     * Controls whether control should use system default provided focus rects
+     *
+     * @platform windows
+     */
+    disableSystemFocusVisuals: PropTypes.bool,
+
+    /**
+     * Called when the view receives focus.
+     *
+     * @platform windows
+     */
+    onFocus: PropTypes.func,
+
+    /**
+     * Called when the view focus is lost.
+     *
+     * @platform windows
+     */
+    onBlur: PropTypes.func,
+
+    /**
+     * Set of keys that should be handled on key down by this component.
+     *
+     * @platform windows
+     */
+    handledKeyDownKeys: PropTypes.arrayOf(PropTypes.number),
+
+    /**
+     * Set of keys that should be handled on key up by this component.
+     *
+     * @platform windows
+     */
+    handledKeyUpKeys: PropTypes.arrayOf(PropTypes.number),
+
+    /**
+     * Called when key down while component has focus.
+     *
+     * @platform windows
+     */
+    onKeyDown: PropTypes.func,
+
+    /**
+     * Called when key up while component has focus.
+     *
+     * @platform windows
+     */
+    onKeyUp: PropTypes.func,
+  };
+
+  static keys = UIManager.WindowsControl.Constants.Keys;
+
+  /**
+   * Splits styles into two sets: one applying to focusable view, and the other one to the embedded child
+   * The only style we care about is "transform"
+   */
+  static splitStyle(styles) {
+      var sets = {};
+
+      sets.focusableStyle = {};
+      sets.childStyle = {};
+
+      if (styles) {
+        var flattenedStyles = flattenStyle(styles);
+
+        for (var styleName in flattenedStyles) {
+          if (styleName == 'transform') {
```

Comment:
>== [](start = 24, length = 2)

Use `====`

Reply:
fixed

---
In reply to: [156484327](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156484327) [](ancestors = 156484327)